### ISSUE TITLE
Support selecting security method in VMESS config

### ISSUE THREAD

**Issue by Vigilans** [v2rayA/v2rayA#622]
### Is the feature related with existing problem?

Currently a configured VMESS server will use `auto` security method, but for VMESS [we also have `none` and `zero`](https://www.v2fly.org/config/protocols/vmess.html#userobject), and `zero` should be a proper option into consideration when configuring VMESS with HTTP2 TLS transport.

### Content of the feature

Add a box for `security` setting for VMESS's `UserObject`.

---

BTW, v2rayA will create a balancer `proxy` instead of outbound if two or more servers are chosen. If a `proxy` balancer could be created even with only one outbound, then it will enable a new way to provide customized server settings through config overriding in multiple config mode (through `-confdir`)

e.g. We have server named `Server`, and v2rayA will create a tag `『Server』` for it. We have some complicated settings for this server that `v2rayA`'s GUI does not provide, then we could write this server config manually in json, and use mutli-conf to override this server setting with the same tag  `『Server』`.

Cite a comment deleted in https://github.com/v2rayA/v2rayA/issues/621, we could just regard `balancer` as a `Outbound Group` that introduces a level of reference:

> (Btw, I wrote aforementioned feature in v2ray-core's config.json as two balancers: international and domestic, and use such balancerTag across routing rules. Here balancerTags just serves as a reference link, or a concept of Outbound Group, by changing the actual outbound in the Outbound Group we control whether domestic and international domains and ips are proxied.)

**Comment by YinAqu** [comment]
Agree, my device is just a router with low cpu resources on which `shellclash` is running, `zero` **may**(not for sure😃) save it's poor cpu.

**Comment by MarksonHon** [comment]
Closed: no activity for 60 days. Reopen or open a new issue if it still happens on the current release.
